docs/controller-skills/euroscope-setup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Euroscope Setup
+---
+
+--8<-- "includes/abbreviations.md"
+
+Initial setup, and future AIRAC Cycle updates must be made manually for the EuroScope client.
+
+VATNZ has two packages available:
+
+  - **SkyLine**: Contains all of the required position and frequency definitions alongside the Sector Files. 
+  - **Sector Files**: Contains only the Sector Files.
+
+This page covers how to install the VATNZ Skyline Package, and how to update the sector files. For a detailed guide on installation of the client itself, refer to the [Euroscope documentation](https://www.euroscope.hu/wp/installation/){target=new}
+
+For most cycles, Controllers will only need to replace the Sector Files. Controllers will be explicitly instructed in the Changelogs should they need to replace the entire SkyLine package.
+
+## VATNZ Skyline Setup
+
+The Skyline package is intended for initial setup of the client, or for larger changes to the dataset. For regular AIRAC cycle updates, please refer to [Updating Sector Files](#updating-sector-files)
+
+!!! warning "Backing up your Settings"
+    Files in the `/Settings/` folder may be overwritten by the new SkyLine package, losing all of your display and list settings. Before updating, we recommend you back-up your `List.txt` and `Screen.txt` files, and reimporting them after you replace the rest of the package.
+
+    It is important that you back-up only those two files, as other files in the `/Settings/` folder would have changed.
+  
+1. [Download the latest SkyLine Package from the VATNZ website](https://www.vatnz.net/airspace/sector_files/){ target=new }. This will download a `.zip` file named "VATNZ-SKYLINE_xxyy", where `xx` is the year, and `yy` is the cycle.
+2. In Windows, open an explorer window and navigate to your EuroScope files. By default this is in the `AppData\Roaming\EuroScope` directory. Note: This may be in the `Documents\EuroScope\` directory for older installations of Euroscope.
```

Comment:
```suggestion
2. In Windows, open an explorer window and navigate to your EuroScope files. By default this is in the `AppData\Roaming\EuroScope` directory. Note: This may be in the `Documents\EuroScope\` directory for older installations of EuroScope.
```